base_multi_company/models/multi_company_abstract.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+
+
+class MultiCompanyAbstract(models.AbstractModel):
+
+    _name = 'multi.company.abstract'
+    _description = 'Multi-Company Abstract'
+
+    company_id = fields.Many2one(
+        string='Company',
+        comodel_name='res.company',
+        compute='_compute_company_id',
+        store=True,
```

Comment:
@lmignon It's the first available, but I see your point. How would just changing to computed solve this though, particularly because it seems like this would be a permissions issue if the user doesn't have access to view the company?